test/module/irohad/network/block_loader_test.cpp

```diff
@@ -104,6 +121,14 @@ class BlockLoaderTest : public testing::Test {
         .transactions(txs);
   }
 
+  void retrieveBlockAndCompare(
+      const shared_model::interface::types::HeightType height,
+      const shared_model::interface::Block &ref_block) {
+    auto retrieved_block =
+        loader->retrieveBlock(peer_key, height).assumeValue();
+    EXPECT_EQ(ref_block, *retrieved_block);
```

Comment:
With this function, we lose which line in the test corresponds to the expectation check. An maybe return `loader->retrieveBlock(peer_key, height).assumeValue()`, and do `EXPECT_EQ` in test body?